analytics.js

```diff
@@ -278,6 +300,7 @@ class Analytics {
         " failed loaded count: ",
         object.failedToBeLoadedIntegration.length
       );
+      leaveBreadcrumb(`Inside replayEvents fn`);
```

Comment:
A better way to put it,
```suggestion
      leaveBreadcrumb(`Start replaying buffered events`);
```